### PR TITLE
reference: concentrate binlog documents

### PR DESCRIPTION
**Pull request by ericsyh**
### What is changed, added or deleted? 

Previously we want to merge the TiDB Binlog documents into TiDB while the effect is not great. So, in this PR, we pull binlog documents back and concentrate again. 

### What is the related PR or file link(s)? 

- #1450 
- https://github.com/pingcap/docs/pull/1251

### Which version does your change affect? 

dev/v3.0/v2.1

### CODE REVIEW

**Comment by GregoryIan** [comment]
LGTM

File: dev/TOC.md [hunk 1]
```diff
@@ -323,7 +316,14 @@
...
     - [TiDB 最佳实践](https://pingcap.com/blog-cn/tidb-best-practice/)
   + [TiSpark 使用指南](/dev/reference/tispark.md)
-  + [TiDB Binlog 简介](/dev/reference/tidb-binlog-overview.md)
+  + TiDB Binlog
+    - [概述](/dev/reference/tools/tidb-binlog/overview.md)
+    - [部署使用](/dev/reference/tools/tidb-binlog/deploy.md)
+    - [运维管理](/dev/reference/tools/tidb-binlog/maintain.md)
+    - [版本升级](/dev/reference/tools/tidb-binlog/upgrade.md)
+    - [监控告警](/dev/reference/tools/tidb-binlog/monitor.md)
+    - [Reparo](/dev/reference/tools/tidb-binlog/reparo.md)
```
Comment (WangXiangUSTC) [inline dev/TOC.md]: 要不改成`增量恢复`？

File: dev/TOC.md [hunk 2]
```diff
@@ -323,7 +316,14 @@
...
+    - [Binlog Slave Client](/dev/reference/tools/tidb-binlog/binlog-slave-client.md)
```
Comment (WangXiangUSTC) [inline dev/TOC.md]: https://github.com/pingcap/docs-cn/blob/master/dev/faq/tidb-binlog.md
也需要移一下 FAQ

**Comment by WangXiangUSTC** [comment]
rest LGTM

File: dev/TOC.md (hunk 2 above)
Comment (ericsyh) [reply dev/TOC.md]: 移到 tidb-binlog folder 下，在 tidb-binlog 和 faq 下都有 tidb-binlog-faq 的 TOC？

File: dev/TOC.md (hunk 1 above)
Comment (ericsyh) [reply dev/TOC.md]: 可以嗯，那 `binlog slave client` 改成 `kafka 同步`？

Comment (WangXiangUSTC) [reply dev/TOC.md]: 说实话我也不知道怎么翻译这个比较好。。。binlog 自定义开发？

File: dev/TOC.md (hunk 2 above)
Comment (lilin90) [reply dev/TOC.md]: @WangXiangUSTC @ericsyh 赞同 faq 下不放。

Comment (yikeke) [reply dev/TOC.md]: 我把 Binlog 的 FAQ 移到 tidb-binlog folder 下了。[commit](https://github.com/pingcap/docs-cn/pull/1946/commits/ea0d31542f2460e5d9eac7941d26dbacc06f7063)

File: dev/reference/tools/tidb-binlog/deploy.md [hunk 3]
```diff
@@ -601,4 +602,4 @@ Drainer="192.168.0.13"
 > - Drainer 不支持对 ignore schemas（在过滤列表中的 schemas）的 table 进行 rename DDL 操作。
 > - 在已有的 TiDB 集群中启动 Drainer，一般需要全量备份并且获取 savepoint，然后导入全量备份，最后启动 Drainer 从 savepoint 开始同步增量数据。
 > - 下游使用 MySQL 或 TiDB 时应当保证上下游数据库的 sql_mode 具有一致性，即下游数据库同步每条 SQL 语句时的 sql_mode 应当与上游数据库执行该条 SQL 语句时的 sql_mode 保持一致。可以在上下游分别执行 `select @@sql_mode;` 进行查询和比对。
-> - 如果存在上游 TiDB 能运行但下游 MySQL 不支持的 DDL 语句时（例如下游 MySQL 使用 InnoDB 引擎时同步语句 `CREATE TABLE t1(a INT) ROW_FORMAT=FIXED;`），Drainer 也会同步失败，此时可以在 Drainer 配置中跳过该事务，同时在下游手动执行兼容的语句，详见[跳过事务](/dev/how-to/maintain/tidb-binlog.md#同步时出现上游数据库支持但是下游数据库执行会出错的-DDL应该怎么办)。
+> - 如果存在上游 TiDB 能运行但下游 MySQL 不支持的 DDL 语句时（例如下游 MySQL 使用 InnoDB 引擎时同步语句 `CREATE TABLE t1(a INT) ROW_FORMAT=FIXED;`），Drainer 也会同步失败，此时可以在 Drainer 配置中跳过该事务，同时在下游手动执行兼容的语句，详见[跳过事务](/dev/reference/tools/tidb-binlog/maintain.md#同步时出现上游数据库支持但是下游数据库执行会出错的-DDL应该怎么办)。
```
Comment (lilin90) [inline dev/reference/tools/tidb-binlog/deploy.md]:
```suggestion
> - 如果存在上游 TiDB 能运行但下游 MySQL 不支持的 DDL 语句时（例如下游 MySQL 使用 InnoDB 引擎时同步语句 `CREATE TABLE t1(a INT) ROW_FORMAT=FIXED;`），Drainer 也会同步失败，此时可以在 Drainer 配置中跳过该事务，同时在下游手动执行兼容的语句，详见[跳过事务](/dev/reference/tools/tidb-binlog/maintain.md#同步时出现上游数据库支持但是下游数据库执行会出错的-ddl应该怎么办)。
```

File: v2.1/reference/tools/tidb-binlog/deploy.md [hunk 4]
```diff
@@ -542,4 +543,4 @@ Drainer="192.168.0.13"
 > - Drainer 不支持对 ignore schemas（在过滤列表中的 schemas）的 table 进行 rename DDL 操作。
 > - 在已有的 TiDB 集群中启动 Drainer，一般需要全量备份并且获取 savepoint，然后导入全量备份，最后启动 Drainer 从 savepoint 开始同步增量数据。
 > - 下游使用 MySQL 或 TiDB 时应当保证上下游数据库的 sql_mode 具有一致性，即下游数据库同步每条 SQL 语句时的 sql_mode 应当与上游数据库执行该条 SQL 语句时的 sql_mode 保持一致。可以在上下游分别执行 `select @@sql_mode;` 进行查询和比对。
-> - 如果存在上游 TiDB 能运行但下游 MySQL 不支持的 DDL 语句时（例如下游 MySQL 使用 InnoDB 引擎时同步语句 `CREATE TABLE t1(a INT) ROW_FORMAT=FIXED;`），Drainer 也会同步失败，此时可以在 Drainer 配置中跳过该事务，同时在下游手动执行兼容的语句，详见[跳过事务](/v2.1/how-to/maintain/tidb-binlog.md#同步时出现上游数据库支持但是下游数据库执行会出错的-DDL应该怎么办)。
+> - 如果存在上游 TiDB 能运行但下游 MySQL 不支持的 DDL 语句时（例如下游 MySQL 使用 InnoDB 引擎时同步语句 `CREATE TABLE t1(a INT) ROW_FORMAT=FIXED;`），Drainer 也会同步失败，此时可以在 Drainer 配置中跳过该事务，同时在下游手动执行兼容的语句，详见[跳过事务](/v2.1/reference/tools/tidb-binlog/maintain.md#同步时出现上游数据库支持但是下游数据库执行会出错的-DDL应该怎么办)。
```
Comment (lilin90) [inline v2.1/reference/tools/tidb-binlog/deploy.md]:
```suggestion
> - 如果存在上游 TiDB 能运行但下游 MySQL 不支持的 DDL 语句时（例如下游 MySQL 使用 InnoDB 引擎时同步语句 `CREATE TABLE t1(a INT) ROW_FORMAT=FIXED;`），Drainer 也会同步失败，此时可以在 Drainer 配置中跳过该事务，同时在下游手动执行兼容的语句，详见[跳过事务](/v2.1/reference/tools/tidb-binlog/maintain.md#同步时出现上游数据库支持但是下游数据库执行会出错的-ddl应该怎么办)。
```

File: v3.0/reference/tools/tidb-binlog/deploy.md [hunk 5]
```diff
@@ -602,4 +602,4 @@ Drainer="192.168.0.13"
 > - Drainer 不支持对 ignore schemas（在过滤列表中的 schemas）的 table 进行 rename DDL 操作。
 > - 在已有的 TiDB 集群中启动 Drainer，一般需要全量备份并且获取 savepoint，然后导入全量备份，最后启动 Drainer 从 savepoint 开始同步增量数据。
 > - 下游使用 MySQL 或 TiDB 时应当保证上下游数据库的 sql_mode 具有一致性，即下游数据库同步每条 SQL 语句时的 sql_mode 应当与上游数据库执行该条 SQL 语句时的 sql_mode 保持一致。可以在上下游分别执行 `select @@sql_mode;` 进行查询和比对。
-> - 如果存在上游 TiDB 能运行但下游 MySQL 不支持的 DDL 语句时（例如下游 MySQL 使用 InnoDB 引擎时同步语句 `CREATE TABLE t1(a INT) ROW_FORMAT=FIXED;`），Drainer 也会同步失败，此时可以在 Drainer 配置中跳过该事务，同时在下游手动执行兼容的语句，详见[跳过事务](/v3.0/how-to/maintain/tidb-binlog.md#同步时出现上游数据库支持但是下游数据库执行会出错的-DDL应该怎么办)。
+> - 如果存在上游 TiDB 能运行但下游 MySQL 不支持的 DDL 语句时（例如下游 MySQL 使用 InnoDB 引擎时同步语句 `CREATE TABLE t1(a INT) ROW_FORMAT=FIXED;`），Drainer 也会同步失败，此时可以在 Drainer 配置中跳过该事务，同时在下游手动执行兼容的语句，详见[跳过事务](/v3.0/reference/tools/tidb-binlog/maintain.md#同步时出现上游数据库支持但是下游数据库执行会出错的-DDL应该怎么办)。
```
Comment (lilin90) [inline v3.0/reference/tools/tidb-binlog/deploy.md]:
```suggestion
> - 如果存在上游 TiDB 能运行但下游 MySQL 不支持的 DDL 语句时（例如下游 MySQL 使用 InnoDB 引擎时同步语句 `CREATE TABLE t1(a INT) ROW_FORMAT=FIXED;`），Drainer 也会同步失败，此时可以在 Drainer 配置中跳过该事务，同时在下游手动执行兼容的语句，详见[跳过事务](/v3.0/reference/tools/tidb-binlog/maintain.md#同步时出现上游数据库支持但是下游数据库执行会出错的-ddl应该怎么办)。
```

Comment (yikeke) [reply v3.0/reference/tools/tidb-binlog/deploy.md]: PTAL again. @lilin90 @lilin90

Comment (lilin90) [reply v3.0/reference/tools/tidb-binlog/deploy.md]: @yikeke This comment for v3.0 hasn't been fixed.

**Comment by lilin90** [comment]
Please also fix my comment for v3.0. Thanks! @ericsyh @yikeke

**Comment by yikeke** [comment]
> Please also fix my comment for v3.0. Thanks! @ericsyh @yikeke

Fixed.

File: v3.0/reference/tools/tidb-binlog/maintain.md [hunk 6]
```diff
@@ -131,6 +132,8 @@ Usage of binlogctl:
 
     {{< copyable "shell-regular" >}}
 
+    {{< copyable "shell-regular" >}}
```
Comment (lilin90) [inline v3.0/reference/tools/tidb-binlog/maintain.md]: Duplicate code snippet with line 133.

**Pull request merged**